src/html/HTMLWriter.jl

```diff
@@ -2395,7 +2395,7 @@ function domify(dctx::DCtx, node::Node, d::Dict{MIME,Any})
         latex = d[MIME"text/latex"()]
         # Make sure to match multiline strings!
         m_bracket = match(r"\s*\\\[(.*)\\\]\s*"s, latex)
-        m_dollars = match(r"\s*\$\$(.*)\$\$\s*"s, latex)
+        m_dollars = match(r"\s*\$\$?(.*?)\$?\$\s*"s, latex)
```

Comment:
```suggestion
        m_dollars = match(r"\s*\$\$?(.*?)\$?\$\s*"s, latex)
```

Wait, why the middle `?`?

Reply:
Because otherwise it will eagerly expand the middle `()` and capture the optional `\$` at the end:
```julia
julia> match(r"\s*\$\$?(.*?)\$?\$\s*"s, "abc \$ x_1 \$ def")
RegexMatch(" \$ x_1 \$ ", 1=" x_1 ")

julia> match(r"\s*\$\$?(.*?)\$?\$\s*"s, "abc \$\$ x_1 \$\$ def")
RegexMatch(" \$\$ x_1 \$\$ ", 1=" x_1 ")

julia> match(r"\s*\$\$?(.*)\$?\$\s*"s, "abc \$ x_1 \$ def")
RegexMatch(" \$ x_1 \$ ", 1=" x_1 ")

julia> match(r"\s*\$\$?(.*)\$?\$\s*"s, "abc \$\$ x_1 \$\$ def")
RegexMatch(" \$\$ x_1 \$\$ ", 1=" x_1 \$")   # <-- wrong
```

Reply:
Ah, makes sense. Let's add this this as a comment though!